pages/common/pest.md

```diff
@@ -0,0 +1,24 @@
+# pest
+
+> A PHP testing framework with a focus on simplicity.
+> More information: <https://pestphp.com>.
+
+- Initialise a standard Pest configuration in the current directory:
+
+`pest --init`
+
+- Run tests in the current directory:
+
+`pest`
+
+- Run tests annotated with the given group:
+
+`pest --group {{name}}`
+
+- Run tests and generate a coverage report in `stdout`:
```

Comment:
I'm not sure about this example description:
```suggestion
- Run tests and print the coverage report to `stdout`:
```

Reply:
Whoops, yeah... it doesn't make sense. 🤦🏻